network/p2p/network.go

```diff
@@ -318,6 +318,18 @@ func (n *Network) UnRegisterChannel(channel channels.Channel) error {
 	return nil
 }
 
+// SendReputationFeedback reports reputation feedback to libP2P for a message received and processed by an message engine. The message engine
+// will report feedBack as true if a message was consumed in some happy path and false if a message was not useful. Currently this
+// fun is a no-op and will log a message.
+func (n *Network) SendReputationFeedback(id peer.ID, messageID string, topic channels.Topic, feedBack bool) {
+	n.logger.Debug().
+		Str("peer_id", id.Pretty()).
+		Str("message_id", messageID).
+		Str("topic", topic.String()).
+		Bool("feedback", feedBack).
+		Msg("sending libP2P reputation feedback for message")
+}
```

Comment:
If the application layer feedback is _negative_, it makes more sense to log at a more severe level, e.g., warn. Also, it adds more observability to have metrics counting the number of negative feedbacks per channel.